test/test_ops.py

```diff
@@ -1519,12 +1519,82 @@ class TestRefsOpsInfo(TestCase):
         '_refs.zeros_like'
     }
 
+    not_in_decomp_table = {
+        # duplicated in _decomp and _refs
+        '_refs.nn.functional.elu',
+        '_refs.masked_fill',
+        '_refs.transpose',
+        '_refs.var',
+        # these are not aten ops?
```

Comment:
Yeah, most of these are actually implemented in Python (e.g., tanhshrink). Actually, this poses a code duplication problem that we should discuss with @mruberry and @ngimel, because in many cases the original implementations of these functions in PyTorch proper would have worked just fine, should we really have two equivalent implementations of a function